cmd/juju/subnet/create.go

```diff
@@ -25,7 +25,7 @@ func NewCreateCommand() modelcmd.ModelCommand {
 type CreateCommand struct {
 	SubnetCommandBase
 
-	CIDR      names.SubnetTag
+	CIDR      string
```

Comment:
Swapping names.SubnetTag for string so that juju compiles and the tests run. This command needs audit and work, per the already created trello card.